gateway/1.8/modules/ROOT/pages/_partials/flex-limits.adoc

```diff
@@ -0,0 +1,24 @@
+//tag::intro[]
+[%header%autowidth.spread,cols=">.<a,a,a"]
+|===
+| Limit | Description | Notes
```

Comment:
My interpretation of the [guidelines](https://confluence.internal.salesforce.com/display/DOCTEAM/Supplemental+Topic+Guidelines#:~:text=The%20standard%20columns,and%20additional%20information) is that the limit is the first column and the value is the second.

You typically describe the thing first before giving the number so that the reader can skim the left column for what they're looking for.